.github/workflows/llama_stack.yml

```diff
@@ -77,10 +77,13 @@ jobs:
           python-version: ${{ matrix.python-version }}
 
       - name: Test Llama Stack Server
+        env:
+          OLLAMA_INFERENCE_MODEL: llama3.2:3b
+          OLLAMA_URL: http://localhost:11434
         shell: bash
         run: |
           pip install uv
-          INFERENCE_MODEL=llama3.2:3b uv run --with llama-stack llama stack build --template ollama --image-type venv --run < /dev/null > server.log 2>&1 &
+          uv run --with llama-stack llama stack build --distro starter --image-type venv --run < /dev/null > server.log 2>&1 &
```

Comment:
They changed `--template` to `--distro` https://github.com/meta-llama/llama-stack/pull/3035

And they combined all basic providers into a `starter` distro (docs [here](https://llama-stack.readthedocs.io/en/latest/distributions/self_hosted_distro/starter.html)) You switch between the different providers by setting specific env variables which I've done above.